apis/api-journeys-modern/src/schema/journeyEventsExportLog/journeyEventsExportLog.spec.ts

```
@@ -4,9 +4,111 @@ import { getClient } from '../../../test/client'
+              }
+            })
+          }
+          return Promise.resolve({ data: null })
+        })
+      }
+      return jest.fn().mockImplementation(() => Promise.resolve({ data: null }))
+    })
+  }
+})
+
 type Journey = Prisma.JourneyGetPayload<{}>
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)